doc/radosgw/metrics.rst

```diff
@@ -152,7 +152,7 @@ Op metrics can also be tracked per-user or per-bucket. These metrics are exporte
         ...
     ]
 
-:ref:`rgw-multitenancy` allows to use buckets and users of the same name simultaneously. If a user or bucket lies under a tenant, a label for tenant in the form Tenant = {tenantid} is added to the metric.
+:ref:`rgw-multitenancy` allows to use buckets and users of the same name simultaneously. If a user or bucket lies under a tenant, a label for tenant in the form ``Tenant = {tenantid}`` is added to the metric.
```

Comment:
S/to/the/

lowercase label

Reply:
I believe you propose "allows the use of buckets [...]"?

Reply:
Pushed these updates, please reply if I interpreted the replacement incorrectly.